syntax = "proto3";

package greet;

service Greeter {
    rpc SayHello(HelloRequest) returns(HelloReply);
    rpc ParrotSayHello(HelloRequest) returns(stream HelloReply);
}
message HelloRequest{
    string greeting = 1;
}

message HelloReply{
    string message = 1 ;
}
